## `nixpkgs-review` result                                                                                                                                                                 
                                                                                                                                                                                           
Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).                                                                                                               
                                                                                                                                                                                           
Command: `nixpkgs-review pr 510728`                                                                                                                                                        
Commit: `80bc8a49a71e3e4404722944679cf402158f7b81`                                                                                                                                         
                                                                                                                                                                                           
---                                                                                                                                                                                        
### `aarch64-linux`                                                                                                                                                                        
<details>                                                                                                                                                                                  
  <summary>:fast_forward: 4 packages marked as broken and skipped:</summary>                                                                                                               
  <ul>                                                                                                                                                                                                                                                                                                                                                                                 
    <li>python313Packages.dvc-gdrive</li>                                                                                                                                                  
    <li>python313Packages.dvc-gdrive.dist</li>                                                                                                                                             
    <li>python314Packages.dvc-gdrive</li>                                                                                                                                                  
    <li>python314Packages.dvc-gdrive.dist</li>                                                                                                                                                                                                                                                                                                                                         
  </ul>                                                                                                                                                                                    
</details>                                                                                                                                                                                 
<details>                                                                                                                                                                                  
  <summary>:x: 118 packages failed to build:</summary>                                                                                                                                                                                                                                                                                                                                 
  <ul>                                                                                                                                                                                     
    <li>dvc (python313Packages.dvc)</li>                                                                                                                                                   
    <li>dvc-with-remotes</li>                                                                                                                                                              
    <li>dvc-with-remotes.dist</li>                                                                                                                                                         
    <li>dvc.dist (python313Packages.dvc.dist)</li>                                                                                                                                         
    <li>froide</li>                                                                                                                                                                        
    <li>froide-govplan</li>                                                                                                                                                                
    <li>froide-govplan.dist</li>                                                                                                                                                           
    <li>froide.dist</li>                                                                                                                                                                   
    <li>inventree</li>                                                                                                                                                                     
    <li>inventree.dist</li>                                                                                                                                                                
    <li>lasuite-docs</li>                                                                                                                                                                  
    <li>lasuite-docs.dist</li>                                                                                                                                                             
    <li>lasuite-meet</li>                                                                                                                                                                                                                                                                                                                                                              
    <li>lasuite-meet.dist</li>                                                                                                                                                                                                                                                                                                                                                         
    <li>mediagoblin</li>                                                                                                                                                                   
    <li>mediagoblin.dist</li>                                                                                                                                                              
    <li>paperless-ngx</li>                                                                                                                                                                                                                                                                                                                                                             
    <li>paperless-ngx.dist</li>                                                                                                                                                                                                                                                                                                                                                        
    <li>pretalx</li>                                                                                                                                                                       
    <li>pretalx.dist</li>                                                                                                                                                                  
    <li>pretalx.static</li>                                                                                                                                                                                                                                                                                                                                                            
    <li>pretix</li>                                                                                                                                                                                                                                                                                                                                                                    
    <li>pretix.dist</li>                                                                                                                                                                   
    <li>python313Packages.celery</li>                                                                                                                                                      
    <li>python313Packages.celery-batches</li>                                                                                                                                              
    <li>python313Packages.celery-batches.dist</li>                                                                                                                                         
    <li>python313Packages.celery-redbeat</li>                                                                                                                                              
    <li>python313Packages.celery-redbeat.dist</li>                                                                                                                                         
    <li>python313Packages.celery-singleton</li>                                                                                                                                            
    <li>python313Packages.celery-singleton.dist</li>                                                                                                                                       
    <li>python313Packages.celery.dist</li>                                                                                                                                                 
    <li>python313Packages.django-celery-beat</li>                                                                                                                                          
    <li>python313Packages.django-celery-beat.dist</li>                                                                                                                                     
    <li>python313Packages.django-celery-email</li>                                                                                                                                         
    <li>python313Packages.django-celery-email.dist</li>                                                                                                                                    
    <li>python313Packages.django-celery-results</li>                                                                                                                                                                                                                                                                                                                                   
    <li>python313Packages.django-celery-results.dist</li>                                                                                                                                  
    <li>python313Packages.django-filingcabinet</li>                                                                                                                                        
    <li>python313Packages.django-filingcabinet.dist</li>                                                                                                                                   
    <li>python313Packages.django-google-analytics-app</li>                                                                                                                                                                                                                                                                                                                             
    <li>python313Packages.django-google-analytics-app.dist</li>                                                                                                                            
    <li>python313Packages.django-health-check</li>                                                                                                                                         
    <li>python313Packages.django-health-check.dist</li>                                                                                                                                    
    <li>python313Packages.django-lasuite</li>                                                                                                                                                                                                                                                                                                                                          
    <li>python313Packages.django-lasuite.dist</li>                                                                                                                                         
    <li>python313Packages.django-raster</li>                                                                                                                                               
    <li>python313Packages.django-raster.dist</li>                                                                                                                                          
    <li>python313Packages.django-structlog</li>                                                                                                                                            
    <li>python313Packages.django-structlog.dist</li>                                                                                                                                       
    <li>python313Packages.djmail</li>                                                                                                                                                      
    <li>python313Packages.djmail.dist</li>                                                                                                                                                 
    <li>python313Packages.dvc-hdfs</li>                                                                                                                                                    
    <li>python313Packages.dvc-hdfs.dist</li>                                                                                                                                               
    <li>python313Packages.dvc-task</li>                                                                                                                                                    
    <li>python313Packages.dvc-task.dist</li>                                                                                                                                               
    <li>python313Packages.dvclive</li>                                                                                                                                                     
    <li>python313Packages.dvclive.dist</li>                                                                                                                                                
    <li>python313Packages.flower</li>                                                                                                                                                      
    <li>python313Packages.flower.dist</li>                                                                                                                                                 
    <li>python313Packages.mmcv</li>                                                                                                                                                        
    <li>python313Packages.mmcv.dist</li>                                                                                                                                                   
    <li>python313Packages.mmengine</li>                                                                                                                                                    
    <li>python313Packages.mmengine.dist</li>                                                                                                                                               
    <li>python313Packages.opentelemetry-instrumentation-celery</li>                                                                                                                        
    <li>python313Packages.opentelemetry-instrumentation-celery.dist</li>                                                                                                                   
    <li>python313Packages.swh-scheduler</li>                                                                                                                                               
    <li>python313Packages.swh-scheduler.dist</li>                                                                                                                                          
    <li>python313Packages.tenant-schemas-celery</li>                                                                                                                                       
    <li>python313Packages.tenant-schemas-celery.dist</li>                                                                                                                                  
    <li>python314Packages.celery</li>                                                                                                                                                      
    <li>python314Packages.celery-batches</li>                                                                                                                                              
    <li>python314Packages.celery-batches.dist</li>                                                                                                                                         
    <li>python314Packages.celery-redbeat</li>                                                                                                                                              
    <li>python314Packages.celery-redbeat.dist</li>                                                                                                                                         
    <li>python314Packages.celery-singleton</li>                                                                                                                                            
    <li>python314Packages.celery-singleton.dist</li>                                                                                                                                       
    <li>python314Packages.celery.dist</li>                                                                                                                                                 
    <li>python314Packages.django-celery-beat</li>                                                                                                                                          
    <li>python314Packages.django-celery-beat.dist</li>                                                                                                                                     
    <li>python314Packages.django-celery-email</li>                                                                                                                                         
    <li>python314Packages.django-celery-email.dist</li>                                                                                                                                    
    <li>python314Packages.django-celery-results</li>                                                                                                                                       
    <li>python314Packages.django-celery-results.dist</li>                                                                                                                                  
    <li>python314Packages.django-filingcabinet</li>                                                                                                                                        
    <li>python314Packages.django-filingcabinet.dist</li>                                                                                                                                   
    <li>python314Packages.django-google-analytics-app</li>                                                                                                                                 
    <li>python314Packages.django-google-analytics-app.dist</li>                                                                                                                            
    <li>python314Packages.django-health-check</li>                                                                                                                                         
    <li>python314Packages.django-health-check.dist</li>                                                                                                                                    
    <li>python314Packages.django-lasuite</li>                                                                                                                                              
    <li>python314Packages.django-lasuite.dist</li>                                                                                                                                         
    <li>python314Packages.django-raster</li>                                                                                                                                               
    <li>python314Packages.django-raster.dist</li>                                                                                                                                          
    <li>python314Packages.django-structlog</li>                                                                                                                                            
    <li>python314Packages.django-structlog.dist</li>                                                                                                                                       
    <li>python314Packages.djmail</li>                                                                                                                                                      
    <li>python314Packages.djmail.dist</li>                                                                                                                                                 
    <li>python314Packages.dvc</li>                                                                                                                                                         
    <li>python314Packages.dvc-hdfs</li>                                                                                                                                                    
    <li>python314Packages.dvc-hdfs.dist</li>                                                                                                                                               
    <li>python314Packages.dvc-task</li>                                                                                                                                                    
    <li>python314Packages.dvc-task.dist</li>                                                                                                                                               
    <li>python314Packages.dvc.dist</li>                                                                                                                                                    
    <li>python314Packages.dvclive</li>                                                                                                                                                     
    <li>python314Packages.dvclive.dist</li>                                                                                                                                                
    <li>python314Packages.flower</li>                                                                                                                                                      
    <li>python314Packages.flower.dist</li>                                                                                                                                                 
    <li>python314Packages.mmcv</li>                                                                                                                                                        
    <li>python314Packages.mmcv.dist</li>                                                                                                                                                   
    <li>python314Packages.mmengine</li>                                                                                                                                                    
    <li>python314Packages.mmengine.dist</li>                                                                                                                                               
    <li>python314Packages.opentelemetry-instrumentation-celery</li>                                                                                                                        
    <li>python314Packages.opentelemetry-instrumentation-celery.dist</li>                                                                                                                   
    <li>python314Packages.swh-scheduler</li>                                                                                                                                               
    <li>python314Packages.swh-scheduler.dist</li>                                                                                                                                          
    <li>python314Packages.tenant-schemas-celery</li>                                                                                                                                       
    <li>python314Packages.tenant-schemas-celery.dist</li>                                                                                                                                  
    <li>swh</li>                                                                                                                                                                           
  </ul>                                                                                                                                                                                    
</details>   
<details>
  <summary>:white_check_mark: 74 packages built:</summary>
  <ul>
    <li>authentik</li>
    <li>netbox</li>
    <li>nixpkgs-manual</li>
    <li>peering-manager</li>
    <li>python313Packages.altcha</li>
    <li>python313Packages.altcha.dist</li>
    <li>python313Packages.dj-rest-auth</li>
    <li>python313Packages.dj-rest-auth.dist</li>
    <li>python313Packages.drf-spectacular</li>
    <li>python313Packages.drf-spectacular.dist</li>
    <li>python313Packages.drf-standardized-errors</li>
    <li>python313Packages.drf-standardized-errors.dist</li>
    <li>python313Packages.netbox-attachments</li>
    <li>python313Packages.netbox-attachments.dist</li>
    <li>python313Packages.netbox-bgp</li>
    <li>python313Packages.netbox-bgp.dist</li>
    <li>python313Packages.netbox-contract</li>
    <li>python313Packages.netbox-contract.dist</li>
    <li>python313Packages.netbox-documents</li>
    <li>python313Packages.netbox-documents.dist</li>
    <li>python313Packages.netbox-floorplan-plugin</li>
    <li>python313Packages.netbox-floorplan-plugin.dist</li>
    <li>python313Packages.netbox-interface-synchronization</li>
    <li>python313Packages.netbox-interface-synchronization.dist</li>
    <li>python313Packages.netbox-napalm-plugin</li>
    <li>python313Packages.netbox-napalm-plugin.dist</li>
    <li>python313Packages.netbox-plugin-prometheus-sd</li>
    <li>python313Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python313Packages.netbox-qrcode</li>
    <li>python313Packages.netbox-qrcode.dist</li>
    <li>python313Packages.netbox-reorder-rack</li>
    <li>python313Packages.netbox-reorder-rack.dist</li>
    <li>python313Packages.netbox-routing</li>
    <li>python313Packages.netbox-routing.dist</li>
    <li>python313Packages.netbox-topology-views</li>
    <li>python313Packages.netbox-topology-views.dist</li>
    <li>python313Packages.translate-toolkit</li>
    <li>python313Packages.translate-toolkit.dist</li>
    <li>python313Packages.translation-finder</li>
    <li>python313Packages.translation-finder.dist</li>
    <li>python313Packages.weblate-language-data</li>
    <li>python313Packages.weblate-language-data.dist</li>
    <li>python314Packages.altcha</li>
    <li>python314Packages.altcha.dist</li>
    <li>python314Packages.dj-rest-auth</li>
    <li>python314Packages.dj-rest-auth.dist</li>
    <li>python314Packages.drf-spectacular</li>
    <li>python314Packages.drf-spectacular.dist</li>
    <li>python314Packages.drf-standardized-errors</li>
    <li>python314Packages.drf-standardized-errors.dist</li>
    <li>python314Packages.netbox-bgp</li>
    <li>python314Packages.netbox-bgp.dist</li>
    <li>python314Packages.netbox-documents</li>
    <li>python314Packages.netbox-documents.dist</li>
    <li>python314Packages.netbox-interface-synchronization</li>
    <li>python314Packages.netbox-interface-synchronization.dist</li>
    <li>python314Packages.netbox-plugin-prometheus-sd</li>
    <li>python314Packages.netbox-plugin-prometheus-sd.dist</li>
    <li>python314Packages.netbox-qrcode</li>
    <li>python314Packages.netbox-qrcode.dist</li>
    <li>python314Packages.netbox-reorder-rack</li>
    <li>python314Packages.netbox-reorder-rack.dist</li>
    <li>python314Packages.netbox-routing</li>
    <li>python314Packages.netbox-routing.dist</li>
    <li>python314Packages.translate-toolkit</li>
    <li>python314Packages.translate-toolkit.dist</li>
    <li>python314Packages.translation-finder</li>
    <li>python314Packages.translation-finder.dist</li>
    <li>python314Packages.weblate-language-data</li>
    <li>python314Packages.weblate-language-data.dist</li>
    <li>tandoor-recipes</li>
    <li>weblate</li>
    <li>weblate.dist</li>
    <li>weblate.static</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python313Packages.celery</summary>
<pre>t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
  /nix/store/cpddva5bzinfkdb7wdkc9xmfx7v8lzrh-python3.13-billiard-4.2.4/lib/python3.13/site-packages/billiard/popen_fork.py:70: DeprecationWarning: This process (pid=737) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

t/unit/worker/test_worker.py: 10 warnings
  /nix/store/cpddva5bzinfkdb7wdkc9xmfx7v8lzrh-python3.13-billiard-4.2.4/lib/python3.13/site-packages/billiard/popen_fork.py:70: DeprecationWarning: This process (pid=769) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

t/unit/concurrency/test_prefork.py: 321 warnings
  /nix/store/cpddva5bzinfkdb7wdkc9xmfx7v8lzrh-python3.13-billiard-4.2.4/lib/python3.13/site-packages/billiard/popen_fork.py:70: DeprecationWarning: This process (pid=726) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED t/unit/backends/test_database.py::test_DatabaseBackend::test_cleanup - sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: ...
= 1 failed, 3223 passed, 34 skipped, 3 xfailed, 404 warnings in 101.30s (0:01:41) =</pre>
</details>
<details>
<summary>python314Packages.celery</summary>
<pre>t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
t/unit/worker/test_worker.py::test_WorkController::test_with_file_descriptor_safety
  /nix/store/b0j9ll2yz3dja4j6ndli7vxk1qjzh3ll-python3.14-billiard-4.2.4/lib/python3.14/site-packages/billiard/popen_fork.py:70: DeprecationWarning: This process (pid=746) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

t/unit/worker/test_worker.py: 10 warnings
  /nix/store/b0j9ll2yz3dja4j6ndli7vxk1qjzh3ll-python3.14-billiard-4.2.4/lib/python3.14/site-packages/billiard/popen_fork.py:70: DeprecationWarning: This process (pid=705) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

t/unit/concurrency/test_prefork.py: 321 warnings
  /nix/store/b0j9ll2yz3dja4j6ndli7vxk1qjzh3ll-python3.14-billiard-4.2.4/lib/python3.14/site-packages/billiard/popen_fork.py:70: DeprecationWarning: This process (pid=769) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED t/unit/backends/test_database.py::test_DatabaseBackend::test_save__restore__delete_group - sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: ...
= 1 failed, 3226 passed, 31 skipped, 3 xfailed, 404 warnings in 118.59s (0:01:58) =</pre>
</details>
</details>

That's a lot of failures, I'll turn to draft for now.

--- a/pkgs/development/python-modules/dj-rest-auth/default.nix
+++ b/pkgs/development/python-modules/dj-rest-auth/default.nix
@@ -27,11 +27,6 @@ buildPythonPackage (finalAttrs: {
     hash = "sha256-eUcve2KPcLjKKWU7AxQEZ0mokP185E43Xjm4b+4hQzA=";
   };
 
-  postPatch = ''
-    substituteInPlace setup.py \
-      --replace-fail "==" ">="
-  '';
-
   build-system = [ setuptools ];
 
   buildInputs = [ django ];
@@ -49,6 +44,7 @@ buildPythonPackage (finalAttrs: {
   };
 
   nativeCheckInputs = [
+    pytest-django
     djangorestframework-simplejwt
     pytestCheckHook
     pytest-django

--- a/pkgs/development/python-modules/django-celery-beat/default.nix
+++ b/pkgs/development/python-modules/django-celery-beat/default.nix
@@ -2,6 +2,7 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch,
 
   # build-system
   setuptools,
@@ -20,17 +21,53 @@
   pytestCheckHook,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage (finalAttrs: {
   pname = "django-celery-beat";
-  version = "2.8.1";
+  version = "2.9.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "celery";
     repo = "django-celery-beat";
-    tag = "v${version}";
-    hash = "sha256-pakOpch5r2ug0UDSqEU34qr4Tz1/mkuFiHW+IOUuGcc=";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-UGKMSXB+Hg865sAk5ePc/noO3eNTr7b3pp7tvNvn1T8=";
   };
+
+  # https://github.com/celery/django-celery-beat/pull/1009
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/dc11bdfa06858409da01f2d407bb68486870a559.patch";
+      hash = "sha256-aK8u2CfmNYFwE9CQvKjnwywVcw8MQfXvev7JYU9kz4E=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/ddcbe8bf2c970a82e87f50c0e9c232b0ae951a85.patch";
+      hash = "sha256-uw3l6CA0lnbF1HnNeSgZ91l1ujkB4V3831Yj+L4i+L0=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/088355dc6531bfffc19272c9de55f4d18606634e.patch";
+      hash = "sha256-6pamaLcHbNUGJMw6nCWeY/C1O58choDbJ1j/4SqDPx4=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/273e2f23edee26404a1691320a69dee280f9639b.patch";
+      hash = "sha256-8wZrDi9GDIF+w5ZZDssJGLG3UyE45xqld95nnG3SV0A=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/69f4a231f30df9ff028f8fc0f67cb115e5d53246.patch";
+      hash = "sha256-74p2XxB9ssWpVokY8XVUjkyyE4+/2lijDYZcZSu9ms0=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/4b8246a44a69e79d7648a9f4658abd8b7937b2b8.patch";
+      hash = "sha256-c1ZJHgnxvOoEWAc9oRkcCLRG7sLLsSER/Dq0H8TIOYU=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/17dd67625f99a0e6dbe7fb779f61f336e7af1730.patch";
+      hash = "sha256-w+MynocR80jUARgvuk5cUIdzmdAYGglvrNXph/8XdBE=";
+    })
+    (fetchpatch {
+      url = "https://github.com/celery/django-celery-beat/commit/03018882f088be83e1c78b4ca657b960b3080081.patch";
+      hash = "sha256-YE121p2aGqoj9mRFSQu+oi+8xaz0QE+CsJHdBB7xwUM=";
+    })
+  ];
 
   pythonRelaxDeps = [ "django" ];
 
@@ -56,18 +93,13 @@ buildPythonPackage rec {
     "t/unit/test_schedulers.py"
   ];
 
-  disabledTests = [
-    # AssertionError: 'At 02:00, only on Monday UTC' != 'At 02:00 AM, only on Monday UTC'
-    "test_long_name"
-  ];
-
   pythonImportsCheck = [ "django_celery_beat" ];
 
   meta = {
     description = "Celery Periodic Tasks backed by the Django ORM";
     homepage = "https://github.com/celery/django-celery-beat";
-    changelog = "https://github.com/celery/django-celery-beat/releases/tag/${src.tag}";
+    changelog = "https://github.com/celery/django-celery-beat/releases/tag/${finalAttrs.src.tag}";
     license = lib.licenses.bsd3;
     maintainers = with lib.maintainers; [ onny ];
   };
-}
+})

--- a/pkgs/development/python-modules/translate-toolkit/default.nix
+++ b/pkgs/development/python-modules/translate-toolkit/default.nix
@@ -9,25 +9,26 @@
   # dependencies
   lxml,
   unicode-segmentation-rs,
-  urllib3,
 
   # optional-dependencies
-  tomlkit,
-
-  # tests
-  aeidon,
   charset-normalizer,
-  cheroot,
   fluent-syntax,
-  gettext,
+  vobject,
   iniparse,
+  rapidfuzz,
   mistletoe,
   phply,
   pyparsing,
-  pytestCheckHook,
+  pyenchant,
+  aeidon,
+  tomlkit,
   ruamel-yaml,
-  syrupy,
-  vobject,
+
+  # tests
+  pytestCheckHook,
+  addBinToPathHook,
+  pytest-xdist,
+  gettext,
 }:
 
 buildPythonPackage (finalAttrs: {
@@ -47,36 +48,40 @@ buildPythonPackage (finalAttrs: {
   dependencies = [
     lxml
     unicode-segmentation-rs
-    urllib3
   ];
 
   optional-dependencies = {
+    chardet = [ charset-normalizer ];
+    fluent = [ fluent-syntax ];
+    ical = [ vobject ];
+    ini = [ iniparse ];
+    levenshtein = [ rapidfuzz ];
+    markdown = [ mistletoe ];
+    php = [ phply ];
+    rc = [ pyparsing ];
+    spellcheck = [ pyenchant ];
+    subtitles = [ aeidon ];
     toml = [ tomlkit ];
+    yaml = [ ruamel-yaml ];
   };
 
   nativeCheckInputs = [
-    aeidon
-    charset-normalizer
-    cheroot
-    fluent-syntax
-    gettext
-    iniparse
-    mistletoe
-    phply
-    pyparsing
     pytestCheckHook
-    ruamel-yaml
-    syrupy
-    tomlkit
-    vobject
-  ];
+    addBinToPathHook
+    pytest-xdist
+    gettext
+  ]
+  ++ lib.concatAttrValues finalAttrs.passthru.optional-dependencies;
 
   disabledTests = [
     # Probably breaks because of nix sandbox
     "test_timezones"
+  ];
 
-    # Requires network
-    "test_xliff_conformance"
+  disabledTestPaths = [
+    # Require pytest-snapshot but there are no snapshots checked in
+    "tests/translate/tools/test_pocount.py"
+    "tests/translate/tools/test_junitmsgfmt.py"
   ];
 
   pythonImportsCheck = [ "translate" ];

